cypress/integration/home/header.spec.js

```diff
@@ -41,9 +41,7 @@ describe("Header links", () => {
 
   pages.forEach((page) => {
     it(`should go to ${page.link}`, () => {
-      cy.get(".osano-cm-save").click();
-
-      cy.get("header .nav-link").contains(page.link).click();
+      cy.get("header .nav-link").contains(page.link).click({ force: true });
```

Comment:
well, this is a dirty hack 😄

Reply:
Yes, I am aware of that 😉 Just wanted to see if the test on github passes this way.